whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -48,15 +48,59 @@ <h4 id='error_tag' class='light warning'></h4>
         {{ vendor_form.story }}
         <p class=" field_text">In Port:</p>
         {{ vendor_form.status }}
-        <h3>Location</h3>
-        <p class="field_text">Street Address*:</p>
+        <h3>Location</h3><br>
+
+    <!-- Code for Map based location inputs start -->
+
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
```

Comment:
We already include Jquery 2.1.1. We really shouldn't be calling in multiple versions of Jquery, things would get very ugly very quickly.